portfolio/src/main/webapp/index.html

```diff
@@ -21,6 +21,8 @@ <h1>My Page</h1>
       <menu> <a href="images.html">Album with photos</a> </menu>
 
       <menu> <a href="memes.html">Just some funny videos I watched recently😁</a> </menu>
+
+      <menu> <a href="piano.html">Piano</a> </menu>
     </div>
   </body>
-</html>
+</html>
```

Comment:
Make sure to keep a newline at the end of the file so this message doesn't show up (this also happens for piano.html).